### ISSUE TITLE
Preference center frequencies has some English-language issues

### ISSUE THREAD

**Issue by StoppingBuck** [mautic/mautic#9093]
## Bug Description
The more details the better...

| Q   | A
| --- | ---
| Mautic version | 3.0.2

This is a very small language-related bug, but the default Preference Center (which recipients get redirected to when pressing Unsubscribe on a message, if it's activated) contains some confusing wording.

This is the default layout in 3.0.2:
![image](https://user-images.githubusercontent.com/8426548/90391106-dca3ff80-e08c-11ea-8de9-2b6c0b7ac22c.png)

The phrase 'Do not contact more than X each days/weeks/months' has a couple of issues:
1. First off, it uses wrong grammar - it's each **day/week/month**
2. Second off, 'Do not contact more than...' seems to me to be a bit unclear in its phrasing. 'Do not contact more than X each day', for example, is not proper English and seems a little confusing. An alternative phrase could be 'Do not message me more than X times each day' or something like that (though admittedly I'm not a native English-speaker myself either...)
3. This might just be me missing something, but it seems like the little calendar widget starts each day on Sunday. I'm in Europe and prefer to have weeks starting on Monday. Is there anywhere in Mautic that this is set?
4. We may want to rename the 'My preferred channel' and 'My segments' part, as chances are that most contacts have no idea what a "Channel" or "Segment" is in this context, as they are Mautic-centric concepts. For example "My preferred channel" could be "I prefer communication via...". I have no idea about what to call the segments part though.
5. Finally, I think a lot of people will be confused about both having 'Contact me through email' AND 'My preferred channel: Email' as choices so maybe we should re-phrase the former too to make it more clear what is actually being asked.

Again, this is very minor, but I felt I should mention it nonetheless. I'll see if I can get a PR started, but otherwise anyone else is welcome to jump in :slightly_smiling_face:

**Comment by npracht** [comment]
Hello @StoppingBuck thanks for this first contribution ! As far as it is "only" about grammar, I think it would be a good way to introduce easily your first PR.

Feel free to ask if you need help to do it.

**Comment by RCheesley** [comment]
Hi @StoppingBuck shout if you need any help with finding the translation strings to make the PR!

**Comment by StoppingBuck** [comment]
@RCheesley Thanks for the nudge, I had indeed gotten sidetracked. I've created PR #9492 to try to address this issue. Going through the five points:

1. I couldn't find any place to adjust the days/weeks/months. AFAICS it's buried deep inside a generic `date` codeword, and it wasn't really a big deal anyway.
2. I rephrased 'Do not contact more than...' to hopefully be a little more intuitive
3. The calendar widget start-on-Sunday thing is its own separate issue. If it persists and I can't find a way to get the widget to start on Monday, I'll open another issue.
4. I rephrased 'My preferred channel' to get away from the term 'Channel' (which I know a lot of our own users would have no idea what is). I kept 'My segments' as it seems to be a pretty accurate and broadly used word after all.
5. As mentioned in the PR notes, even with the re-phrasing I'm still not really satisfied with this one. The two phrases still sound way too familiar, and may confuse the average used. Still, I personally believe the new phrase is an improvement.
